sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -219,12 +224,18 @@ statement
     | unsupportedHiveNativeCommands .*?                                #failNativeCommand
     ;
 
+timezone
+    : STRING
+    | {!double_quoted_identifiers}? DOUBLEQUOTED_STRING
+    | LOCAL
+    ;
+
 configKey
     : quotedIdentifier
```

Comment:
Can `configKey` be double-quoted id?

Reply:
Sure, why not? quotedIdentifier does include double-quoted (when enabled).